static/app/views/discover/table/cellAction.tsx

```diff
@@ -26,6 +26,13 @@ export enum Actions {
   RELEASE = 'release',
   DRILLDOWN = 'drilldown',
   EDIT_THRESHOLD = 'edit_threshold',
+  COPY_TO_CLIPBOARD = 'copy_to_clipboard',
+}
+
+export function stringifyValue(value: string | number | string[] | undefined) {
+  if (!value) return '';
+  if (typeof value === 'string') return value;
+  return value.toString();
```

Comment:
`toString` for an array will return the contents of the array, so `[1,2,3].toString()` -> `"1,2,3"`. This might not be what the user expects.

Instead I'd have a check that basically does if primitive call `value.toString()`, otherwise call `JSON.stringify` (falling back to `toString` if  `JSON.stringify` fails).

We don't want to always call `JSON.stringify` because it will add quotes unnecessarily.